### ISSUE TITLE
improve reddit workflow

### ISSUE THREAD

**Issue by aappling-usgs** [USGS-VIZLAB/vizlab#258]
This is currently all manual. Things we could do to improve how the reddit sharing happens:
* add the thumbnail rules for reddit to the vizlab thumbnail publisher
* implement a system for building a good thumbnail png/gif/gifv for reddit - might be resolved by #247 or #248 
* anything else? see https://www.reddit.com/r/dataisbeautiful/comments/71xuoo/hurricane_marias_intensity_illustrated_animated/ for example post

**Comment by jread-usgs** [comment]
One thing to add (probably out of scope for this package) is to pre-populate the post text in markdown format. We usually fill out a g-doc in advance of the different social campaigns, but since reddit uses markdown format (especially useful for links), we should use that in our post drafts. That probably lives elsewhere, but am attaching to this issue because I don't know where else that type of information goes presently.

**Comment by aappling-usgs** [comment]
Yeah, I was thinking about that need last week, too (also applies to the viz caption text), but also couldn't figure out where it would live...will make another issue to capture this thought.
